### ISSUE TITLE
[IMP] not overwrite migration issue line unless the previous PR is closed

### ISSUE THREAD

**Issue by legalsylvain** [OCA/oca-github-bot#194]
See : https://github.com/OCA/oca-github-bot/pull/191#issuecomment-1173127811

@pedrobaeza Said : 
"my initial intention was to always overwrite, thinking that maintainers will know what to do, but now seeing in perspective, it seems that people don't look too much, so that's why I say to change the initial intention to prevent such change unless the PR is closed"

**Comment by sbidoul** [comment]
@legalsylvain can we close this one?

**Comment by legalsylvain** [comment]
No. https://github.com/OCA/oca-github-bot/pull/191#event-7005929252 didn't changed the bot feature regarding this topic. 
Still todo.

**Comment by SirPyTech** [comment]
@legalsylvain this should have been fixed by #218, can you please check?

**Comment by legalsylvain** [comment]
thanks !

**Comment by SirPyTech** [comment]
👍 Thanks for checking and closing
